PURL-SPECIFICATION.rst

```diff
@@ -415,6 +415,7 @@ To parse a ``purl`` string in its components:
   - Discard any empty segment from that split
   - Percent-decode each segment
   - UTF-8-decode each segment if needed in your programming language
+  - Signal an error if any segment contains a solidus `/`
```

Comment:
Nice catch, consistent with the requirement that "When percent-decoded, a segment . . . must not contain a '/' . . . ."  :+1:    

I suggest changing "Signal" to "Return" and, to be consistent with the other references in the spec, changing "solidus" to "slash".

Reply:
Maybe "report" is a better replacement for "signal" than "return" -- the core spec already uses "report" (`is_invalid: a boolean flag set to true if the test should report an error`) and does not use "signal".  And "solidus" does not appear anywhere in the core spec or RFC3986 while "slash" is used in both.

Reply:
Fixed in https://github.com/package-url/purl-spec/pull/455/commits/7a979329bc9e8289b238144b3157c0f9f4a53379